Refine HTTP Server (#13156)
JabRef/jabref: docs/code-howtos/http-server.md

## Start http server

### Starting with IntelliJ

The class starting the server is located in the project `jabsrv-cli` and is called `org.jabref.http.server.cli.ServerCli`.

Test files to server can be passed as arguments.
If that list is also empty, the file `src/main/resources/org/jabref/http/server/http-server-demo.bib` is served.

### Starting with JBang

> In case one wants to interact with the http server only and does not want to setup or run IntelliJ, [JBang](https://www.jbang.dev/download/) can be used.

Reviewer: I'd say would sound even more better
```suggestion
If you only want to interact with the HTTP server and do not want to set up or run IntelliJ, you can use [JBang](https://www.jbang.dev/download/).
```

DEBUG: Server started.
```

## Served libraries

The last opened libraries are served.
`demo` serves Chocolate.bib.
Additional libraries can be served by passing them as arguments.

## Developing with IntelliJ

> IntelliJ Ultimate offers a Markdown-based http-client. One has to open the file `jabsrv/src/test/rest-api.http`.

Reviewer: ```suggestion
IntelliJ Ultimate offers a Markdown-based http-client. You need to open the file`jabsrv/src/test/rest-api.http`.
```

Already using "you" in other sentences this would make it more consistent.
